source/index.md

```diff
@@ -511,6 +511,9 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
   </dd>
 </dl>
 
+<p class="alert alert-info">
+<b>Notice:</b> A capture cannot be made after 180 days after the authorization.
```

Comment:
My eyes stumple upon "after" and "after". Maybe "A capture can only be made until 180 days after the authorization."?